### PR TITLE
Revert "cirrus-ci: disable the FreeBSD 13 builds"

### PR DESCRIPTION
**Pull request by bagder**
This reverts commit 691b71be930f0e285c8f7a76efd56bbe0576cda6.

See #5028

### CODE REVIEW

**Comment by bagder** [comment]
The CI job on FreeBSD 13 timed out twice and is still red. I'm not going to merge until we see it green.

File: .cirrus.yml [hunk 1]
```diff
@@ -51,7 +51,6 @@ task:
         13.0*) SKIP_TESTS='!1242 !1243 !2002 !2003';;
         12.1*) SKIP_TESTS='!1242 !1243 !2002 !2003';;
         11.3*) SKIP_TESTS='!504 !1242 !1243 !2002 !2003';;
```
Comment (mback2k) [inline .cirrus.yml]: You could turn those ! into ~ while you are at it.

Comment (bagder) [reply .cirrus.yml]: But why? We don't have anything on cirrus that shows/cares about ignore tests.

Comment (mback2k) [reply .cirrus.yml]: I think the ignore feature is not only for the reporting stuff, but also to see the error output in the CI logs.

**Pull request closed without merging**